docs/app/Examples/elements/Button/Variations/ButtonSocialExample.js

```diff
@@ -1,36 +1,29 @@
 import React, { Component } from 'react'
-import { Button, Icon } from 'stardust'
+import { Button } from 'stardust'
 
 export default class ButtonSocialExample extends Component {
   render() {
     return (
       <div>
-        <Button className='facebook'>
-          <Icon className='facebook' />
+        <Button className='facebook' icon='facebook'>
```

Comment:
I almost want to put these button social color classes into the color options:

```js
<Button color='facebook' />
```

Thoughts?